extensions/amp-google-assistant-assistjs/0.1/libs/compile.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# install google closure library:  `npm install google-closure-library` at amphtml
+# folder
+
+COMPILER=../../../../node_modules/@ampproject/google-closure-compiler-linux/compiler
```

Comment:
I think we can use `npx google-closure-compiler` to run this on any OS?

Reply:
Yes you are right. Removed the compiler path and replace the existing compile command with "npx google-closure-compiler".